src/AspNetCoreModuleV2/AspNetCore/src/aspnetcore_shim_config.cpp

```diff
@@ -7,63 +7,36 @@
 #include "hostfxr_utility.h"
 #include "ahutil.h"
 
-ASPNETCORE_SHIM_CONFIG::~ASPNETCORE_SHIM_CONFIG()
-{
-}
-
 HRESULT
 ASPNETCORE_SHIM_CONFIG::Populate(
     IHttpServer      *pHttpServer,
     IHttpApplication *pHttpApplication
 )
 {
-    STACK_STRU(strHostingModel, 300);
-    HRESULT                         hr = S_OK;
+    STACK_STRU(strHostingModel, 10);
```

Comment:
wait why only a size of 10?

Reply:
That isn't large enough for outofprocess.

Reply:
haha, being able to count wasn't in my resume

Reply:
Next offense will require a test.